[2024-08-12T09:07:16.598+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T09:07:16.611+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T07:58:00+00:00 [queued]>
[2024-08-12T09:07:16.616+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T07:58:00+00:00 [queued]>
[2024-08-12T09:07:16.616+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-12T09:07:16.622+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): import_weather_data_to_csv> on 2024-08-12 07:58:00+00:00
[2024-08-12T09:07:16.625+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2248) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-12T09:07:16.625+0000] {standard_task_runner.py:64} INFO - Started process 2250 to run task
[2024-08-12T09:07:16.626+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather', 'import_weather_data_to_csv', 'scheduled__2024-08-12T07:58:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpyplc6bmv']
[2024-08-12T09:07:16.628+0000] {standard_task_runner.py:91} INFO - Job 147: Subtask import_weather_data_to_csv
[2024-08-12T09:07:16.655+0000] {task_command.py:426} INFO - Running <TaskInstance: weather.import_weather_data_to_csv scheduled__2024-08-12T07:58:00+00:00 [running]> on host b0294fd9f5f9
[2024-08-12T09:07:16.723+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather' AIRFLOW_CTX_TASK_ID='import_weather_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T07:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-12T07:58:00+00:00'
[2024-08-12T09:07:16.726+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T09:07:16.728+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-12T09:07:16.729+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/weather_data_ingestion.py --date 2024-08-12']
[2024-08-12T09:07:16.736+0000] {subprocess.py:86} INFO - Output:
[2024-08-12T09:07:19.299+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-12T09:07:19.301+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T09:07:19.320+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather, task_id=import_weather_data_to_csv, run_id=scheduled__2024-08-12T07:58:00+00:00, execution_date=20240812T075800, start_date=20240812T090716, end_date=20240812T090719
[2024-08-12T09:07:19.347+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T09:07:19.366+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T09:07:19.367+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
